### DISCUSSION TITLE
Wrong dimensions with .goo file (Elegoo Saturn 3 Ultra)

### DISCUSSION THREAD

**Discussion by molnija3d** [sn4k3/UVtools#781] [Q&A]
Hello! I found a strange problem and can't understand what's wrong.
I generated exposure test with UV_tools. (single exposure, because of multiexposure do not work correctly).
In UV_tool all is looking good - circles are circles.
But when i open this .goo in chitubox circles became an ellipse.
![2023-11-03_21-00-45](https://github.com/sn4k3/UVtools/assets/110596780/ad8fed81-ac1a-4dee-b956-16020265d5b2)
![2023-11-03_21-00-03](https://github.com/sn4k3/UVtools/assets/110596780/a9815970-80e5-463d-b16d-e53271b99d52)
[ultra_48um_.zip](https://github.com/sn4k3/UVtools/files/13253335/ultra_48um_.zip)
And when i load it to printer it print an ellipse! 
What can i do in this situation?
Thank you.

**Comment by sn4k3** [comment]
Nothing you can do. It's expected behavior with circles on UVtools, because your printer uses rectangle pixels instead of even square. 
Your printer uses 19 x 24 um pixel size which have distinct size for width and height (Not a square), while lines and rectangles shapes can have different side sizes, circles on UVtools have only a radius size, so UVtools use the most pixel size as radius.

While it does not produce a perfect circle it does not invalidate the test.

**Reply by Cuzzman77** [reply]
I am having an issue with the exposure time finder displaying very tiny calibrations
![UV tools error](https://github.com/sn4k3/UVtools/assets/137768611/b99a3563-37e3-4e2c-86ec-f2be077580eb)
 and I checked how it looks on my printer and its small there too. I have a saturn 3 ultra 12k

**Reply by sn4k3** [reply]
The main purpose of the test is to validate the smallest detail, to conclude which is the smallest detail it can print at given exposure.
If you enlarge everything it kinda defeat the test, as larger blobs will overcure a bit and "always" print.
Having a magnifier helps in conclusions. 
If you still want to enlarge, use Millimeter unit instead pixel and it will scale acordingly

**Reply by Cuzzman77** [reply]
I understand, but the size is far too small, it prints a third of the size of what it should. I've seen how big it prints on a smaller machine and that is just a little smaller than a credit card. This prints at about 20mm

**Comment by SpackoG** [comment]
Hi,

I‘ve the same issue on my S3u. I changed to mm but the text of the exposure time is that small that it couldn‘t read.
Also the exposure test should arrange the single tests in order to two lines and 3 Rows to get 6 perfect slices of the specific exposure time.

My suggestion to fix this is, to divide the printer resolution accordingly to the amount of exposure times the user defines…
In my opinion 6 or 8 times should perfect. More test are not needed to find a very good time.

Thank you very much for that great tool. Very cool stuff…

Best regards Guido

**Reply by sn4k3** [reply]
The exposure time text has no adjustment setting, in future I can get the size from the "Text" section configuration.

While there is no master scale setting you can run the test as default and then run: tool - resize with 150% or 200%. 

About divide in rows you can do it by using the Left/right margin setting, you will have to try/error with the value, but it will do what you want. Just use the `CTRL + SHIFT + Z` to adjust that setting between attempts. Example:

![image](https://github.com/sn4k3/UVtools/assets/113281/7b1f3487-7773-4139-8695-bb8476d754d8)


This test was made to validate pixels, at which amount your printer can print them safety and at which space they will start to fuse or not, at given exposure time and your temperature conditions. Resize them will invalidate this checking features.

For any validation test I recommend a magnifier, like: https://s.click.aliexpress.com/e/_DkTorXL
As details are getting better, we expect that each pixel is getting smaller.

**Comment by SpackoG** [comment]
Hi,

Thank you for the quick answer.

I'll play a bit alo g your sugestions.


Mit freundlichen Grüßen / Best regards

Guido


Tiago Conceição ***@***.***> schrieb am Fr., 29. Dez. 2023,
19:08:
